.github/workflows/check.yaml

```diff
@@ -0,0 +1,63 @@
+name: Run checks
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  golangci-lint:
```

Comment:
Please, use a concrete version of go and golangci-lint to avoid broken CI on update of the Go or the linter.